helpers/crypto/crypto.go

```diff
@@ -7,16 +7,20 @@ import (
 	"encoding/base64"
 	"errors"
 	"io"
+	"log"
 	"os"
 )
 
 var block cipher.Block
 
 func init() {
+	if os.Getenv("SYMMETRIC_KEY") == "" {
+		log.Println("SYMMETRIC_KEY env unassigned. Unsubscribe link will not work!")
```

Comment:
Maybe we should only say "Environment variable SYMMETRIC_KEY is not assigned." -- we might use it for more than just unsubscribe links.